lib/index.js

```diff
@@ -972,12 +972,8 @@ class Hatch {
 
     _create_hatch_archive() {
         const tarballPath = `${this._path}.tar.gz`;
-        return new Promise((resolve, reject) => {
-            targz().compress(this._path, tarballPath, err => {
-                if (err) { return reject(err); }
-                resolve(tarballPath);
-            });
-        });
+        return tar.c({gzip: true, file: tarballPath}, [this._path])
```

Comment:
I'd take these options and put them in their own var for readability

Reply:
thanks @sgnn7 I added a fixup and rebased.